logisland-plugins/logisland-elasticsearch-plugin/pom.xml

```diff
@@ -92,38 +88,123 @@
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>org.immutables.tools</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>4</version>
+                <version>${shade-plugin.version}</version>
                 <executions>
                     <execution>
                         <phase>package</phase>
                         <goals>
                             <goal>shade</goal>
                         </goals>
                         <configuration>
+                            <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
+
                             <artifactSet>
-                                <excludes>
-                                    <exclude>com.hurence.logisland:*</exclude>
-                                </excludes>
                                 <includes>
-                                    <include>*:*</include>
+                                    <include>com.hurence.logisland:logisland-elasticsearch-plugin</include>
+                                    <include>org.apache.lucene:*</include>
+                                    <include>com.google.guava:*</include>
+                                    <include>com.twitter:*</include>
+                                    <include>com.tdunning:*</include>
+                                    <include>com.spatial4j:*</include>
+                                    <include>com.carrotsearch:*</include>
+                                    <include>org.hdrhistogram:HdrHistogram</include>
+                                    <include>org.elasticsearch:*</include>
+                                    <include>com.thoughtworks.paranamer:*</include>
+                                    <include>org.elasticsearch:*</include>
+                                    <include>org.elasticsearch.client:*</include>
+                                    <include>org.elasticsearch.plugin:*</include>
+                                    <include>org.elasticsearch.test:*</include>
+                                    <include>com.fasterxml.jackson.core:*</include>
+                                    <include>com.fasterxml.jackson.dataformat:*</include>
+                                    <include>io.netty:*</include>
+                                    <include>org.yaml:*</include>
+                                    <include>org.apache.httpcomponents:*</include>
+                                    <include>commons-codec:*</include>
+                                    <include>org.jboss.netty:*</include>
```

Comment:
oups! markdown formatting make difficult the reading of previous  diff. So basically In my environment I simply add in the include section : 

 <include>com.ning:*</include>